RadiationForce_using_FIR_Filter/wecSimInputFile.m

```diff
@@ -0,0 +1,78 @@
+%% Simulation Data
+simu = simulationClass();               % Initialize Simulation Class
+simu.simMechanicsFile = 'RM3.slx';      % Specify Simulink Model File
+simu.mode = 'normal';                   % Specify Simulation Mode ('normal','accelerator','rapid-accelerator')
+simu.explorer = 'off';                   % Turn SimMechanics Explorer (on/off)
+simu.startTime = 0;                     % Simulation Start Time [s]
+simu.rampTime = 100;                    % Wave Ramp Time [s]
+simu.endTime = 1000;                     % Simulation End Time [s]
+simu.solver = 'ode4';                   % simu.solver = 'ode4' for fixed step & simu.solver = 'ode45' for variable step 
+simu.dt = 0.1; 							% Simulation time-step [s]
+
+
+%% Wave Information 
+% % noWaveCIC, no waves with radiation CIC  
+% waves = waveClass('noWaveCIC');       % Initialize Wave Class and Specify Type  
+
+if enable_convolution == 0 && enable_FIR == 0 && enable_ss == 0
+simu.FIR = 0;    
+% Regular Waves  
+waves = waveClass('regular');           % Initialize Wave Class and Specify Type                                 
+waves.height = 2.5;                     % Wave Height [m]
+waves.period = 8;    % Wave Period [s]
```

Comment:
@salhus Suggest changing wave period as there seems to be some unknown phenomena at 8 s that is causing larger than expected differences at long simulation times. We will flag and explore next fiscal year to identify root cause. Propagate this through the other options.

Reply:
- done